low-level/files/opt/seagate/sspl/bin/consuldump.py

```diff
@@ -27,6 +27,8 @@
 
 class ConsulDump():
 
+    Err_string = '500 No cluster leader'
```

Comment:
We need to explore and improve this to catch the exception from consul instead of this hardcoded string which might change with consul version change.

Reply:
I am not sure about version. @indrajitzagade  Will have more idea on this. 
Will move this.